config/default.yml

```diff
@@ -76,6 +76,9 @@ linters:
   MultilineScript:
     enabled: true
 
+  NoPlaceholders:
+    enabled: false
+
```

Comment:
I'm basing this decision mostly on the fact that `AltText` (another HTML accessibility rule) has `enabled: false`.